docs/sources/dashboards/assess-dashboard-usage/index.md

```diff
@@ -106,3 +106,7 @@ You can sort the dashboards by:
 - Views 30 days
 
 {{< figure src="/static/img/docs/enterprise/improved-search-7-5.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Usage insights dashboards
```

Comment:
Already discussed during pairing - just to suggest linking here form the "contents" bullets at the top of the page.